pkg/asset/installconfig/alibabacloud/validation.go

```diff
@@ -73,12 +81,23 @@ func validateMachinePool(client *Client, ic *types.InstallConfig, fldPath *field
 			zones = zones[:3]
 		}
 
+		response, err := client.DescribeAvailableZoneByInstanceType(pool.InstanceType)
```

Comment:
This change does not seem related to adding support for existing VPC, vSwitches, or private zones.

Reply:
Is this change related to this PR? If not, please move it to a new PR.